backend/coins/btc/coin.go

```diff
@@ -46,6 +46,7 @@ type Coin struct {
 	code                  coinpkg.Code
 	name                  string
 	unit                  string
+	formatUnit            string
```

Comment:
docstrings for these two explaining what each of them are for would be useful

Reply:
Done :+1: